rero_ils/modules/libraries/api.py

```diff
@@ -74,6 +74,12 @@ class Library(IlsRecord):
     fetcher = library_id_fetcher
     provider = LibraryProvider
 
+    def get_organisation(self):
+        """Get Organisation."""
+        from ..organisations.api import Organisation
+        organisation_pid = self.replace_refs()['organisation']['pid']
+        return Organisation.get_record_by_pid(organisation_pid)
```

Comment:
delete line 80 and
`return Organisation.get_record_by_pid(self.organisation_pid)`

Reply:
done